res/translations/strings.de.po

```diff
@@ -189,7 +152,7 @@ msgstr "DEFAULT"
 
 msgctxt "spellinfo"
 msgid "create_roqf"
-msgstr "Die berühmte Bardin Miriam bhean'Meddaf war bekannt für ihr außergewöhnliches Geschick mit der Harfe. Ihre Finger sollen sich so schnell über die Saiten bewegt haben, das sie nicht mehr erkennbar waren. Dieser Zauber, der recht einfach in einen Silberring zu bannen ist, bewirkt eine um das zehnfache verbesserte Geschicklichkeit und Gewandheit der Finger. (Das soll sie auch an anderer Stelle ausgenutzt haben, ihr Ruf als Falschspielerin war berüchtigt). Handwerker können somit das zehnfache produzieren, und bei einigen anderen Tätigkeiten könnte dies ebenfalls von Nutzen sein."
+msgstr "Die berühmte Bardin Miriam bhean'Meddaf war bekannt für ihr außergewöhnliches Geschick mit der Harfe. Ihre Finger sollen sich so schnell über die Saiten bewegt haben, das sie nicht mehr erkennbar waren. Dieser Zauber, der recht einfach in einen Silberring zu bannen ist, bewirkt eine um das zehnfache verbesserte Geschicklichkeit und Gewandheit der Finger. (Das soll sie auch an anderer Stelle ausgenutzt haben; ihr Ruf als Falschspielerin war berüchtigt). Handwerker können somit das zehnfache produzieren, und bei einigen anderen Tätigkeiten könnte dies ebenfalls von Nutzen sein."
```

Comment:
Hier kann ich leider in github nicht sehen, was sich geändert hat, weil die Zeile unheimlich lang ist (590+ Zeichen). Einmal automatisch alle .po Dateien auf maximal 80 Zeichen bringen, wäre ein guter erster Schritt für dieses Projekt.
Kann man den gettext Werkzeugen irgendwie sagen, dass man eine maximale Zeilenlänge einhalten will? Zumindest msgattrib hat ein Kommandozeilenargument -w, mit dem man die page width definieren kann.